### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by KSSWSept20**

### CODE REVIEW

**Review by ziiqii — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -1,14 +1,9 @@
...
+* The project simulates an ongoing software project for a desktop application (called _CulinaryContacts_) specially designed for family-owned restaurant owners or managers to manage contact details.
   * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
   * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+* For the detailed documentation of this project, see the **[CulinaryContacts Product Website](https://nus-cs2103-ay2324s2.github.io/tp/)**.
```
Comment (Tanzhiheng26) [inline README.md:9]: I think the website should be: https://ay2324s2-cs2103t-w09-3.github.io/tp/

**Review by Tanzhiheng26 — CHANGES_REQUESTED** [review]
Just a few nits to fix. Can consider changing color of supplier tag in mockup so that it is easier to see.

File: README.md (hunk 1 above)
Comment (KSSWSept20) [reply README.md]: Thanks, changed to this.

**Review by Tanzhiheng26 — APPROVED** [review]
LGTM

**Pull request merged**